knowledge_repo/app/templates/post_editor_base.html

```diff
@@ -560,7 +563,7 @@ <h2> {{ comments | length }} Comments </h2>
   });
 </script>
 <script type="text/javascript"
-  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.8/MathJax.js">
+  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js">
```

Comment:
is this an intentional change to downgrade this library?

Reply:
sorry no. I'll update back